CHANGELOG.rst

```diff
@@ -1,10 +1,10 @@
 Releases
 ========
 
-0.3.4 (unreleased)
+0.4.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added a ``validate`` method to all ``Spec`` types.
```

Comment:
Added just to ``TypeSpec``s. There are a few specs in there that aren't TypeSpecs.